src/ReaderThemeSupportFeatures.php

```diff
@@ -385,4 +490,90 @@ public function get_relative_luminance_from_hex( $hex ) {
 		$lum = ( 0.2126 * $red ) + ( 0.7152 * $green ) + ( 0.0722 * $blue );
 		return (int) round( $lum );
 	}
+
+	/**
+	 * Checks a string for a unit and value and returns an array
+	 * consisting of `'value'` and `'unit'`, e.g. array( '42', 'rem' ).
+	 *
+	 * Copied from `wp_get_typography_value_and_unit()`
+	 *
+	 * @see https://github.com/WordPress/WordPress/blob/9caf1c4adeddff2577c24d622ebbbf278a671271/wp-includes/block-supports/typography.php#L297
+	 *
+	 * @since 2.4.1
+	 *
+	 * @param string|int|float $raw_value Raw size value from theme.json.
+	 * @param array            $options   {
+	 *     Optional. An associative array of options. Default is empty array.
+	 *
+	 *     @type string   $coerce_to        Coerce the value to rem or px. Default `'rem'`.
+	 *     @type int      $root_size_value  Value of root font size for rem|em <-> px conversion. Default `16`.
+	 *     @type string[] $acceptable_units An array of font size units. Default `array( 'rem', 'px', 'em' )`;
+	 * }
+	 * @return string|null The value and unit, or null if the value is empty.
+	 */
+	private function get_typography_value_and_unit( $raw_value, $options = [] ) {
```

Comment:
We can ignore coverage for this copied method?

Reply:
Sure